src/libgap-api.h

```diff
@@ -197,4 +197,15 @@ extern Int GAP_ValueOfChar(Obj obj);
 // Returns the GAP character object with value <obj>.
 extern Obj GAP_CharWithValue(UChar obj);
 
+// Assigns the value of the GAP machine float object <obj>
```

Comment:
Nitpick: There should be a section break comment before this, i.e.,
```C
////
//// floats
////
```
and ideally this section then should be sorted between the "calls" and "integers" section.